packages/synapse-react-client/src/components/DataGrid/SynapseGrid.tsx

```diff
@@ -125,14 +162,7 @@ const SynapseGrid = forwardRef<SynapseGridHandle, SynapseGridProps>(
       [modelSnapshot, schemaPropertiesInfo],
     )
 
-    const commit = useCallback(
-      throttle(() => {
-        console.log('Auto-committing changes')
-        // The model has already been updated in handleChange, just send the patch
-        websocketInstance?.sendPatch()
-      }, 500),
-      [websocketInstance],
-    )
+    const commit = useThrottledSendPatch(websocketInstance)
```

Comment:
split this into its own function with improved logic for handling debouncing. After reconnecting, kept getting console errors stating the websocket was closed when attempting to send messages.